cypress/selectors/isecodes/isecodesSelectors.js

```diff
@@ -1,5 +1,5 @@
 const selectors = {
   isecodesList: '[data-test-subcase-ise-codes-list]',
-  isecodesListItem: '[data-test-subcase-ise-codes-list-item]',
+  isecodesListItem: '[data-test-subcase-ise-codes-agenda-overview-item]',
```

Comment:
Deze change klopt niet. Die selector heeft niets te maken met die overview

'data-test-subcase-ise-codes-agenda-overview-item' heeft geen verdere ref in de code buiten in deze selector
'data-test-subcase-ise-codes-list-item' wordt gebruikt in 
```
app/pods/components/agenda/agendaitem/agendaitem-case/subcase-ise-codes/template.hbs
```

Reply:
fixed it